src/ngraph/op/util/broadcasting.hpp

```diff
@@ -125,5 +125,38 @@ namespace ngraph
                 new_shape,
                 calculate_broadcast_axes(new_shape, node->get_shape(), start_match_axis));
         }
+
+        /// \brief      Makes a Constant nGraph node, broadcasing to a specified shape if needed.
+        ///
+        /// Current implementation supports broadcasting a scalar value to the desired shape.
+        ///
+        /// \param[in]  type   The Constant node element type.
+        /// \param[in]  shape  The tensor data shape.
+        /// \param[in]  data   The data to initialize node with.
+        ///
+        /// \tparam     T      Input data value type.
+        ///
+        /// \return     The nGraph node representing Constant data.
+        ///
+        template <typename T>
+        std::shared_ptr<ngraph::Node> make_constant_node(const ngraph::element::Type& type,
+                                                         const ngraph::Shape& shape,
+                                                         const std::vector<T>& data)
```

Comment:
Can you add it here instead? https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/builder/make_constant.hpp

Reply:
I'll do you one better: we can remove `make_constant_node` all together and just use `make_constant` instead.